### DISCUSSION TITLE
What to do with the Repology rate-limit workaround preset?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#30746] [Suggest an Idea]
### Intro

We have a [`workarounds:reduceRepologyServerLoad`](https://docs.renovatebot.com/presets-workarounds/#workaroundsreducerepologyserverload) preset. The description for the preset mentions issue #10133, which we closed a long time ago.

I think we have these options:

- Remove preset, _if_ we're confident Renovate behaves well now.
- Link to #27802 (and maybe remove the preset when that lands).
- Something else.

### History

The plan was to remove the Repology workaround preset when we closed these issues:

- #10133
- #9550

I think the conclusion we reached was: `429` retry is included in the `got` package so Renovate should listen to incoming `429` responses.

But the Repology maintainer said:

> For the note, I'm not seeing any noticeable decrease in 429s since this was deployed

I'm not sure if we actually fixed the "Renovate ignores rate-limit messages from Repology servers" problem?

### Link to newer issue instead?

We have a new issue, about setting hard limits for Repology:

- #27802

Maybe we should link to that issue instead. When that issue is done, we should check again if we need to keep the workaround.

### Or something else?

Or maybe you have a better idea. :wink:

**Comment by rarkins** [comment]
@zharinov can you add this as a default rate limit so that the workarounds preset is no longer necessary?

**Reply by rarkins** [reply]
https://github.com/renovatebot/renovate/issues/30991

**Reply by zharinov** [reply]
I did it earlier, but turns out it was in my imagination, i.e. without any real PR. Here we go: https://github.com/renovatebot/renovate/pull/30996
